web/src/components/profile/ProfileNaicsCode.tsx

```diff
@@ -27,13 +29,34 @@ export const ProfileNaicsCode = (): ReactElement => {
     return `/tasks/${urlSlug}`;
   }, [roadmap]);
 
+  const lockField = (): boolean => {
+    return (
+      state.profileData.naicsCode !== "" &&
+      (state.profileData.operatingPhase === "UP_AND_RUNNING" ||
+        state.profileData.operatingPhase === "UP_AND_RUNNING_OWNING")
```

Comment:
shouldn't this be based on whether or not tax access was success or not? This is just saying the field is locked for all Oscars and all up-and-running Poppys, always?